tests/data/scad/3D/features/linear_extrude-tests.scad

```diff
@@ -19,6 +19,10 @@ translate([19,20,0]) linear_extrude(height=20, twist=45, slices=10) square([10,1
 
 translate([0,-15,0]) linear_extrude(5) square([10,10]);
 
+translate([15,-15,0]) linear_extrude(v=[3 ,4 ,5]) square([10, 10]);
+
+translate([30,-15,0]) linear_extrude(height=10, v=[3, 4, 5]) square([10,10]);
```

Comment:
Since unspecified components of `v` in linear_extrude defaults to zero, we should test 1- and 2- dimensional v values as well.